internal/tools/securitypolicy/main.go

```diff
@@ -172,8 +184,61 @@ func createPolicyFromConfig(config Config) (sp.SecurityPolicy, error) {
 			container.Layers = append(container.Layers, hashString)
 		}
 
+		// add rules for all known environment variables from the configuration
+		// these are in addition to "other rules" from the policy definition file
+		config, err := img.ConfigFile()
+		if err != nil {
+			return p, err
+		}
+		for _, env := range config.Config.Env {
+			rule := sp.SecurityPolicyEnvironmentVariableRule{
+				Strategy: "string",
+				Rule:     env,
+			}
+
+			container.EnvRules = append(container.EnvRules, rule)
+		}
+
+		// cri adds TERM=xterm for all workload containers. we add to all containers
```

Comment:
This kinda worries me if this is ever changed. I'm still not sure how this even always get's set from our investigation. For now, not sure what else you could do at the moment so not saying to change

Reply:
this tool is a convenience for developers working on gcs and policy, if it changes, it is easy enough to adjust. this merely saves having to put it in every toml for that someone has.

Reply:
I need to check where the code I'm reviewing lives before I type. This is fine then